README.md

```diff
@@ -136,6 +136,28 @@ client.report.custom(fields, format = "JSON")
 client.report.find(report_number, format = "JSON", fd = true)
 ```
 
+### Applicant Tracking System
+
```

Comment:
Can you add a comment letting people know that the ATS is still in beta? In case they make incompatible changes

Reply:
👍